client/src/components/GraphViewerComponent/GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent.js

```diff
@@ -565,10 +566,34 @@ export class GraphViewerCytoscapeComponent extends React.Component {
     });
   }
 
+  saveSessionLayout() {
```

Comment:
I think we need to rename this method. It should probably be something like `saveSessionLayoutIfNoneExists`.

Reply:
Done